blackbox/utils/reports.py

```diff
@@ -40,7 +42,8 @@ def success(self) -> bool:
     @property
     def output(self) -> str:
         """Return the combined outputs from all the database reports."""
-        return "\n".join(report.output for report in self.databases)
+        return self.sanitize_output(
+            "\n".join(report.output for report in self.databases))
```

Comment:
There's no reason to change this line in this pull request. It's only 89 characters long, which is perfectly acceptable. This only hurts readability. Please configure your automatic code formatter to be a little bit less zeleous, and to use at least 100 as its maximum allowed line length.